frontend/src/app/pages/marker-page/marker-page.component.scss

```diff
@@ -45,7 +53,10 @@ button {
         }
 
         .active-button {
-            background-color: #fb8c00;
+            background-color: mat-color($accent, 600);
```

Comment:
Should these active buttons have white color?

It looks weird for me:
<img width="304" alt="screenshot 2018-11-11 at 21 42 48" src="https://user-images.githubusercontent.com/6514553/48318051-2085ed00-e5fb-11e8-9743-b29235104f68.png">